README.md

```diff
@@ -93,10 +93,18 @@ blank. Once done, you should be able to ping the robot at `192.168.131.1`.
 
 ## Usage
 
+URDF files of the robots are used for kinematics and simulation. Compile the
```

Comment:
Note that in Python the xacro files can be directly compiled programmatically using xacrodoc (this is done e.g. in `kinematics.py`). Probably no need to mention that here though.